.buildkite/pipelines/ci_integrations_pipeline.yml

```diff
@@ -13,22 +18,19 @@ steps:
       - cd metrics/ci_integrations
       - pub get
       - pub run test_coverage --print-test-output --no-badge
-    artifact_paths: "metrics/ci_integrations/coverage/lcov.info"
-
-  - label: "Export coverage report"
-    key: "coverage"
-    depends_on: "tests"
-    commands:
-      - curl -o coverage_converter -k https://github.com/platform-platform/monorepo/releases/download/coverage-converter-snapshot/coverage_converter_linux -L
-      - chmod a+x coverage_converter
-      - buildkite-agent artifact download metrics/ci_integrations/coverage/lcov.info .
-      - ./coverage_converter lcov -i metrics/ci_integrations/coverage/lcov.info -o coverage-summary.json
-    artifact_paths: "coverage-summary.json"
+      - coverage_converter lcov -i coverage/lcov.info -o coverage-summary.json
```

Comment:
I'm wondering if this step uses the latest release of the `coverage_converter` tool? For example, when we publish a new snapshot for `coverage_converter` it seems like we should build a new version of `testmnrp/dart-converter` to ensure it uses the last release. The same case we have for the `ci_integrations` tool and `testmnrp/sync-builddata`. Do we need to handle this somehow or maybe introduce a new infrastructure to automate updating images with the new tools snapshots?